meetings/2025-01-29.md

```diff
@@ -0,0 +1,87 @@
+
+* elf Pavlik
+
+## Topics
+
+### [New Work Item] ActivityPub/Fediverse interoperability
+
+https://github.com/solid/specification/issues/708
+
+* eP: It is just an announcement, next week I hope to bring it up for a vote.
+* PAC: It is importantn to have people from Social Web CG also involved. We need consistent story about Solid and Activity Pub. 
+* HZ: Question to PAC, isn't it more important to bring it to LWS WG?
+* PAC: The charter mentions Social Web CG as a contact that we will synchronize with. There is no active Social Web WG. WGs have active scope, they need to sync, what we are discussin isn't directly in the scope of the LWS WG.
+* HZ: What is the best path forward to bring it to LWS
+* PAC: This is less urgent for the WG, at this moment it focuses on use cases. I don't expect strong coupling with Social Web CG. This work item should be a joined work in my opinion. Hopefully it will progress in CGs and we will have better understanding how to progress.
+* RG: The way we were discussin it yesterday, it will be layer on top of Solid, not something in the Solid spec. Everything should be workable as a layer on top of solid. 
+
+### State of CG reports
+
+https://www.w3.org/2024/09/linked-web-storage-wg-charter.html#dependencies
+
+> Depending on the Working Group progress, including consideration for adequate implementation experience, the Group may also decide to adopt the following dependencies as input documents
+
+* eP: It sounds like Solid-OIDC, ACP, WAC and Solid Notifications **MAY** be adopted by the WG. Where new implementation/deployment feedback should be addressed meanwhile?
+* eP: Possible inspiration for CG-WG coordination: https://github.com/w3c-fedid/Administration/blob/main/proposals-CG-WG.md
+* PAC: WG says that at this moment those reports are not formally considered as inputs from CGs. I would consider it ok to continue working on those reports in CG. WG may at some point signal that it wants to take over a specific report.
+* PAC: As for incubation in FedCM community group, I assume that this is browser centric and it may relate to how the browser vendors work. It might not transfer well in our community.
+* eP: ...
+* RG: I have a larger criticism of the process. The whole requirement that you need to be an invited expert or paid member to prarticipate. For example I was refused to even observe the WG meetings. I think this might be a problem.
+* eP: ...
+* RG: The minutes give some idea. Aaron asked me to raise use cases that I mentioned during the TPAC. When I asked to observe the relevant meeting I was refused.
+* PAC: It is inaccurate to say that non IE or members are not allowed. Chairs can invite observers to the meeting. I wasn't aware of your request. The process allows it and it would make sense if the use case you propossed is going to be discussed. On the other hand having dozens of observers on every meeting would not work. I will come back to the chairs to better understand your specific case.
```

Comment:
```suggestion
* RG: The minutes give some idea. Aaron asked me to raise use cases that I mentioned during the TPAC. When I asked to observe the relevant meeting, I was refused.
* PAC: It is inaccurate to say that non-IE or non-members are not allowed. Chairs can invite observers to any meeting. I wasn't aware of your request. The process allows it, and it would make sense for you to be there if the use case you proposed is going to be discussed. On the other hand, having dozens of observers at every meeting would not work. I will follow up with the chairs to better understand your specific case.
```